cmd/ferretdb/main.go

```diff
@@ -217,12 +221,97 @@ func main() {
 	case "run":
 		run()
 	case "ping":
-		// TODO https://github.com/FerretDB/FerretDB/issues/4246
+		ping()
 	default:
 		panic("unknown sub-command")
 	}
 }
 
+// ping creates connection to FerretDB instance specified by the flags, and runs `ping` command against it.
```

Comment:
For new authentication without  `--setup-database`, is it expected to skip ping? I think that should be documented.

Reply:
done